### DISCUSSION TITLE
Missing Fuel in MSP telemetry

### DISCUSSION THREAD

**Discussion by zosko** [iNavFlight/inav#7405] [Q&A]
Hello, i did not found how to get **Fuel** over MSP telemetry, Is it not included or miss it ?
Seems is locked MSPv1 for improvements but i also dont see in MSPv2. 
Can someone tell me how to can extract from MSPv1 if is possible if not i can use MSPv2.

**Comment by DzikuVx** [comment]
What do you mean `Fuel`? INAV has no support for fuel sensor

**Reply by zosko** [reply]
@DzikuVx i mean Fuel percentage, like on Smart Port i receive like 0 to 100% of battery presented like Fuel percentage like here
https://github.com/iNavFlight/inav/blob/7d24ebd5f75470526f0fba45b890789454b6b36a/src/main/telemetry/telemetry.h#L50

Im develop iNav Telemetry app for iOS devices

https://github.com/zosko/iNavTelemetry
